runtime_test.go

```diff
@@ -2384,6 +2416,79 @@ func TestStringToBytesConversion(t *testing.T) {
 	}
 }
 
+// func TestPromiseAll(t *testing.T) {
```

Comment:
[q] Were these tests commented out intentionally? They seem uncommented on [master](https://github.com/dop251/goja/blob/5937a312edda9e86e50a49da61ad09fde83063c3/runtime_test.go#L2574)

Reply:
Yeah which makes me think I should add a comment, basically we don't want to use goja Promises but for diffing reasons I keep the tests and the code just commenting it out.

Reply:
SGTM! I think a comment could be helpful just in case but commenting out makes sense